sdk/textanalytics/Azure.AI.TextAnalytics.Legacy.Shared/src/GeneratorStubs/InternalTypes.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Legacy.Models
```

Comment:
There is precedent for this pattern in Search and Storage.

Reply:
These types will not need to be linked into the core Text Analytics client library - they exist only to influence code generation.